support/Environments/minerva_clang.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/env sh
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Instructions to **compile** spectre on Minerva:
+#
+# 1. Clone the spectre repository, if you haven't already. A good place is in
+#    your `/home` directory on Minerva:
+#    ```sh
+#    cd /home/yourname
+#    git clone git@github.com:yourname/spectre.git
+#    ```
+# 2. Set the `$SPECTRE_HOME` environment variable to the location of the spectre
+#    repository, e.g. `/home/yourname/spectre`:
+#    ```sh
+#    export SPECTRE_HOME=path/to/spectre
+#    ```
+# 3. Source this script and setup modules:
+#    ```sh
+#    source path/to/this/script.sh`
+#    spectre_setup_modules
+#    ```
+#    Note: Add steps 2 and 3 to your `.bashrc` file if you don't want to repeat
+#    them every time you log in. The `spectre_setup_modules` function only
+#    adjusts your `MODULEPATH` to make the installed modules visible but doesn't
+#    load any, so it's safe to add to your `.bashrc`.
+# 4. Create a build directory, if you don't have one already. A good place is in
+#    your `/work` directory on Minerva:
+#    ```sh
+#    cd /work/yourname
+#    mkdir spectre-build
+#    ```
+#    Note: Add a timestamp or descriptive labels to the name of the build
+#    directory name, since you may create more build directories later, e.g.
+#    `build_YYYY-MM-DD` or `build-clang-Debug`.
+# 5. Run `cmake` to configure the build directory:
+#    ```sh
+#    cd path/to/build/directory
+#    module purge
+#    spectre_run_cmake
+#    ```
+#    Note: Remember to `module purge` to work in a clean environment, unless you
+#    have reasons not to.
+# 6. Compile! With the build directory configured, this script sourced and
+#    modules set up, you can skip the previous steps from now on.
+#    ```sh
+#    module purge
+#    spectre_load_modules
+#    make -j16 ExportCoordinates1D
+#    ```
+#
+# Instructions to **run** spectre executables on Minerva:
+#
+# 1. Create a run directory. A good place is in your `/scratch` directory on
+#    Minerva. Make sure to choose a descriptive name, e.g.
+#    `/scratch/yourname/spectre/your_project/00_the_run`.
+# 2. Copy `support/SubmitScripts/Minerva.sh` to the run directory and edit it as
+#    the comments in that file instruct.
+# 3. Submit the job to Minerva's queue:
+#    ```sh
+#    sbatch Minerva.sh
+#    ```
+
+spectre_setup_modules() {
+    export MODULEPATH="\
+/home/SPACK2021/share/spack/modules/linux-centos7-haswell:$MODULEPATH"
+    export MODULEPATH="\
+/home/nfischer/spack/share/spack/modules/linux-centos7-haswell:$MODULEPATH"
```

Comment:
here and in the other script this will make your directory the preferred search path for modules over the SPACK2021 dir. Is this intended?

Reply:
yea this is just a Spack install tree that I manage, which provides modules that I have built for spectre. The _order_ of module search paths doesn't really matter here because the Spack modules are identified uniquely by a hash.